Working-Groups.md

```diff
@@ -1,235 +1,37 @@
-# Planetary Software Organization Top Level Working Groups
+# Planetary Software Organization Top-Level Working Groups
 
-Planetary Software Organization Top Level Working Groups are autonomous projects created by the
-[Technical Steering Committee (TSC)][].
+Planetary Software Organization Top-Level Working Groups (TLWGs)
+are autonomous projects typically initiated by the [Technical
+Steering Committee (TSC)](TSCcharter), but external individuals can
+also submit an application to form a TLWG.
 
-Working Groups can be formed at any time but must be ratified by the TSC.
-Once formed the work defined in the Working Group charter is the
-responsibility of the WG rather than the TSC.
+Top-Level Working Groups can be formed at any time but must go
+through the TLWG [lifecycle process](lifecycle) and be approved
+by the TSC.  Once formed, the work defined in the TLWG charter is
+the responsibility of the TLWG rather than the TSC.
 
-It is important that Working Groups are not formed pre-maturely. Working
-Groups are not formed to *begin* a set of tasks but instead are formed
-once that work is already underway and the contributors
-think it would benefit from being done as an autonomous project.
+It is important that TLWGs are not formed pre-maturely. TLWGs are
+not typically formed to *begin* a set of tasks but instead are
```

Comment:
I don't think we should open this up to an atypical situation. TLPs and TLWGs should cover work that has already begun before they are formalized under the PS TSC.

Reply:
Again, I think you're drawing too-bright a line here.  What does 'already begun' mean in this context.  I don't think we need to define it sharply.  The process that we have outlined allows TSC members and mentors the opportunity to weigh in on whether a particular TLP or TLWG application is 'enough' to get started.

Reply:
I would rather have that flexibility in the acceptance process than in the documents themselves. I would like the document to set the requirement and then the acceptance process decide if the TLP or TLWG meets the requirement.

I think the following lines adequately describe what 'already begun' means, the work is happening but as part of a different project or in an ad hoc fashion.

Reply:
"typically" has been removed.